### PR TITLE
Update documentation (link Clinica paper, ClinicaDL, typos, black badge)

### PR DESCRIPTION
**Pull request by alexandreroutier**

### CODE REVIEW

**Review by ghisvail — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,16 +23,20 @@
...
+  <a href="https://github.com/psf/black">
+    <img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Code style: black">
+  </a>
 </p>
 
 <p align="center">
   <a href="http://www.clinica.run">Homepage</a> |
   <a href="https://aramislab.paris.inria.fr/clinica/docs/public/latest/">Documentation</a> |
-  <a href="https://hal.inria.fr/hal-02308126">Preprint</a> |
+  <a href="https://doi.org/10.3389/fninf.2021.689675">Paper</a> |
```
Comment (omar-rifai) [inline README.md:34]: J'ai une erreur "DOI not found" pour ce lien

File: README.md [hunk 2]
```diff
@@ -149,12 +151,13 @@ for more information.
 
 ## Citing us
 
-- Routier, A., Burgos, N., Díaz, M., Bacci, M., Bottani, S., El-Rifai O., Fontanella, S., Gori, P., Guillon, J., Guyot, A., Hassanaly, R., Jacquemont, T.,  Lu, P., Marcoux, A.,  Moreau, T., Samper-González, J., Teichmann, M., Thibeau-Sutre, E., Vaillant G., Wen, J., Wild, A., Habert, M.-O., Durrleman, S., and Colliot, O.: ‘Clinica: An Open Source Software Platform for Reproducible Clinical Neuroscience Studies’, 2021. [hal-02308126](https://hal.inria.fr/hal-02308126)
-
+- Routier, A., Burgos, N., Díaz, M., Bacci, M., Bottani, S., El-Rifai O., Fontanella, S., Gori, P., Guillon, J., Guyot, A., Hassanaly, R., Jacquemont, T.,  Lu, P., Marcoux, A.,  Moreau, T., Samper-González, J., Teichmann, M., Thibeau-Sutre, E., Vaillant G., Wen, J., Wild, A., Habert, M.-O., Durrleman, S., and Colliot, O.:
+*Clinica: An Open Source Software Platform for Reproducible Clinical Neuroscience Studies* Frontiers in Neuroinformatics, 2021
+[doi:10.3389/fninf.2021.689675](https://doi.org/10.3389/fninf.2021.689675)
```
Comment (omar-rifai) [inline README.md:156]: idem DOI introuvable, est-ce que c'est juste une question de temps?

Comment (ghisvail) [reply README.md:156]: je pense, l'article est toujours en "provisionally accepted" sur Frontiers

File: docs/CAPS/Introduction.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ Several differences exist between the BIDS and CAPS specifications.
 
 - CAPS assumes that the session is always present in a BIDS dataset even though there is a single session. In other words, all datasets are considered longitudinal, even when they have only one session.
 
-- NifTi files generated by a Clinica pipeline are always compressed, while this is not systematic with the BIDS specifications.
+- NIfTI files generated by a Clinica pipeline are always compressed, while this is only recommended by the BIDS specifications.
```
Comment (omar-rifai) [inline docs/CAPS/Introduction.md:44]: je ne comprend pas la phrase "while this is only recommended ..", est-ce que ce ne serais pas plutôt "[..] compressed, this is recommended by .."

File: docs/Converters/ADNI2BIDS.md [hunk 4]
```diff
@@ -301,7 +301,7 @@ clinica convert adni-to-bids -h
 !!! cite "Example of paragraph:"
     The ADNI data have been curated and converted to the Brain Imaging Data Structure (BIDS) format
     [[Gorgolewski et al., 2016](https://doi.org/10.1038/sdata.2016.44)] using Clinica
-    [[Routier et al.](https://hal.inria.fr/hal-02308126/);
+    [[Routier et al., 2021](https://doi.org/10.3389/fninf.2021.689675);
```
Comment (omar-rifai) [inline docs/Converters/ADNI2BIDS.md:304]: idem

**Review by omar-rifai — COMMENTED** [review]
Hello @alexandreroutier, thanks for updating this. I seem to have a problem with the doi you provided in all the changes. Maybe it just temporary?

File: docs/CAPS/Introduction.md (hunk 3 above)
Comment (ghisvail) [reply docs/CAPS/Introduction.md:44]: He wants to say Clinica forces compression of NIfTI files while this is only a recommendation in the BIDS spec.

Comment (ghisvail) [reply docs/CAPS/Introduction.md:44]: "systematic" is not idiomatic English and reads as word-to-word translated from French

Comment (ghisvail) [reply docs/CAPS/Introduction.md:44]: Tbh we could drop this part of the sentence entirely.

Comment (omar-rifai) [reply docs/CAPS/Introduction.md:44]: Granted for "systematic"  but I still think the phrasing is a bit awkward. I would change it to something of the sort "[..] are always compressed. Compression is  "recommended" but not compulsory*  in the BIDS specifications" 

*or the appropriate BIDS term

Comment (omar-rifai) [reply docs/CAPS/Introduction.md]: > Tbh we could drop this part of the sentence entirely.

Also works for me

File: README.md (hunk 2 above)
Comment (omar-rifai) [reply README.md:156]: Si vous êtes serein par rapport au fait que c'est le bon DOI, OK

**Review by omar-rifai — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ghisvail) [reply README.md:156]: C'est celui qui est sur [Frontiers](https://www.frontiersin.org/articles/10.3389/fninf.2021.689675) en tout cas.

**Comment by alexandreroutier** [comment]
> Hello @alexandreroutier, thanks for updating this. I seem to have a problem with the doi you provided in all the changes. Maybe it just temporary?

Hi,

The DOI is not yet activated. Once the paper will be [published](https://www.frontiersin.org/articles/10.3389/fninf.2021.689675) - it is currently "Provisionally accepted", this should work on the same time (best case scenario) or a couple of days after (worst case scenario).

File: docs/Atlases.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ It was built on anatomical MRI of 24 healthy subjects from which 74 cortical ROI
 It is possible to run the [`t1-volume`](../T1_Volume) and [`pet-volume`](../PET_Volume) <!--and [`dwi-dti`](../DWI_DTI)--> pipelines using a custom parcellation.
 To do so:
 
-- Install Clinica following the [developer instructions](../../Installation/#install-clinica);
+- Install Clinica following the [developer instructions](../Installation/#install-clinica);
```
Comment (omar-rifai) [inline docs/Atlases.md:81]: In that link we only ask for installation through `pip install` and not through the source code. Was there another page or shouldn't we add that part in the documentation?

Comment (omar-rifai) [reply docs/Atlases.md:81]: My bad the developer instructions are there but hidden by default in an expandable box.

Comment (ghisvail) [reply docs/Atlases.md:81]: See #358

**Comment by omar-rifai** [comment]
Looks good, I'll merge! Thanks for the updates.

**Pull request merged**